src/model/Series.ts

```diff
@@ -558,6 +569,14 @@ class SeriesModel<Opt extends SeriesOption = SeriesOption> extends ComponentMode
         }
 
         const data = this.getData(dataType);
+        if ((data.getItemModel(dataIndex) as Model).get(['select', 'disabled']) === true) {
+            return false;
+        }
```

Comment:
It seems this logic isn't necessary as if a data item disables `select` state, the `selectedMap` won't contain it. (See `_innerSelect` function). Correct me if I'm missing something.

Reply:
I think https://github.com/apache/echarts/pull/15534#discussion_r780309973 can explain it.

Reply:
I think https://github.com/apache/echarts/pull/15534#discussion_r780309973 can explain it.

BTW we usually don't check `true` explicitly unless the value can have other types than boolean. Check `truthy` is more commonly used for boolean only value.
```ts
if (...get(['select', 'disabled']) { ... }
```